cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Protos/cmsAdapter.proto

```diff
@@ -528,6 +530,24 @@ service DocumentManager {
 	rpc GetDocumentSubTypeGuid(DocumentIdRequest) returns (GetDocumentSubTypeIdReply);
     rpc GetDmer(CaseIdRequest) returns (GetDmerReply);
     rpc GetDriverAndCaseDocuments(GetDriverAndCaseDocumentsRequest) returns (GetDriverAndCaseDocumentsReply);
+    
+  // Update Claim DMER on documents
+    rpc UpdateClaimDmer(UpdateClaimRequest) returns (UpdateClaimReply);
+    rpc UpdateUnClaimDmer(UpdateClaimRequest) returns (UpdateClaimReply);
+ 
+}
+
+message UpdateClaimRequest{
+  string loginId = 1;
+  string caseId= 2;
+  string driverId = 3;
```

Comment:
you don't need caseId or driverId here

Reply:
I have just added them if we need them in future while handling multiple login ids

Reply:
I don't see why you would need them. In general, it's better not to add "just in case" code. It's better to add it when you actually need it.

Reply:
Agree Jebb. Just deleted them we can add when we need them.